genai_stack/memory/langchain.py

```diff
@@ -0,0 +1,30 @@
+from langchain.memory import ConversationBufferMemory as cbm
+from .base import BaseMemory
+
+class ConversationBufferMemory(BaseMemory):
+    memory = None
```

Comment:
You should create a data model and the config class instead of initialising in the class variables. Check embedding or other components on how to do it.